docs/0023_sensitive_columns_exposed.md

```diff
@@ -0,0 +1,110 @@
+
+Level: ERROR
+
+### Rationale
+
+Tables exposed via the Supabase Data APIs that contain columns with potentially sensitive data (such as passwords, SSNs, credit card numbers, API keys, or other PII) pose a significant security risk when Row Level Security (RLS) is not enabled. Without RLS, anyone with access to the project's URL and an anonymous or authenticated role can read all data in these tables, potentially exposing sensitive user information.
```

Comment:
>pose a significant security risk when Row Level Security (RLS) is not enabled.

I think this one is already covered by the "RLS not enabled" rule

What might be more relevant is if these columns are accessible to anonymous users (they should only be accessible to authenticated users)